collector/src/main/java/com/bitmovin/analytics/data/DRMInformation.kt

```diff
@@ -0,0 +1,3 @@
+package com.bitmovin.analytics.data
+
+data class DRMInformation(var loadTime: Long? = null, var type: String? = null)
```

Comment:
do we need both props as nullable ? I think instead in Adapter whole `DRMInformation` could be nullable instead.

Reply:
https://github.com/bitmovin/bitmovin-analytics-collector-android/pull/80/commits/784c55f4d297a7a6ab20bbfe1f0017164ffb8875
needed to make more changes to accomplish that, but i think it is better this way 👍